src/components/pulp-download-button.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import DownloadIcon from '@patternfly/react-icons/dist/esm/icons/download-icon';
+import { Button } from '@patternfly/react-core';
+import { downloadString } from "src/utilities";
+
+export const PulpDownloadButton = ({
+  text,
+  filename,
+}: {
+  text: string;
+  filename: string;
+}) => {
+
+  return (
+    <Button
+      variant='plain'
+      aria-label={filename + '-download'}
```

Comment:
```suggestion
      aria-label={t`Download ${filename}`}
```
?

(aria-label is supposed to be human-readable)

(+ `import { t } from '@lingui/macro';`)